src/controllers/beneficial.owner.type.controller.ts

```diff
@@ -15,24 +15,17 @@ import { isActiveFeature } from '../utils/feature.flag';
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `GET ${config.BENEFICIAL_OWNER_TYPE_PAGE}`);
-
     const appData: ApplicationData = getApplicationData(req.session);
-
     const hasTrusts: boolean = checkEntityHasTrusts(appData);
 
     logger.infoRequest(req, `${config.BENEFICIAL_OWNER_TYPE_PAGE} hasTrusts=${hasTrusts}`);
 
     return res.render(config.BENEFICIAL_OWNER_TYPE_PAGE, {
       backLinkUrl: config.BENEFICIAL_OWNER_STATEMENTS_URL,
       templateName: config.BENEFICIAL_OWNER_TYPE_PAGE,
+      submitUrl: config.BENEFICIAL_OWNER_TYPE_SUBMIT_URL,
       hasTrusts,
       ...appData,
-      pageParams: {
```

Comment:
I'm wondering why trusts related code had to change - at this point, why is this block removed?

Reply:
No longer needed it was linked to the old anchor tag:


 {% if (hasTrusts) %}
            {% set nextPage = pageParams.urlToTrust %}
          {% else %}
            {% set nextPage = pageParams.urlToCheckYourAnswers %}
          {% endif %}

          <a href= "{{ nextPage }}"
             role="button"
             draggable="false"
             class="govuk-button govuk-button--secondary govuk-!-margin-top- 6"
             data-module="govuk-button"
             id="submit"
          >No more <span class="govuk-visually-hidden"> {{ hiddenTextValue }} </span> to add</a>